frontend/src/views/HomeView.vue

```diff
@@ -17,16 +31,184 @@ const { getConfig } = storeToRefs(useConfigStore());
+            images and documents.
+          </p>
+        </div>
+      </div>
+      <div class="col-6 pl-5">
+        <div class="flex align-items-center">
+          <img
+            src="@/assets/images/home_2.png"
+            class="border-1 drop-shadow"
+          />
+        </div>
+      </div>
+    </div>
+    <div class="grid mb-8">
+      <div class="col-6 pr-5">
+        <div class="flex align-items-left mb-2">
+          <h1 class="font-bold">
+            Manage access and share
+          </h1>
+        </div>
+        <div class="flex align-items-left">
+          <p class="text-xl">
+            You can assign custom permissions to other users through IDIR or BCeID authentication.<br /><br />
+            You can also choose to share files with public links.
+            Contact your organization's privacy and security teams to see if this fits your intended use.
+            Make sure you don't upload personal or private information without the consent of your Ministry
+            Privacy Officer.
+          </p>
+        </div>
+      </div>
+      <div class="col-6 pl-5">
+        <div class="flex">
+          <img
+            src="@/assets/images/home_3.png"
+            class="border-1 drop-shadow"
+          />
+        </div>
+      </div>
+    </div>
+    <div class="flex justify-content-center mb-5">
+      <h1 class="font-bold">
+        Versioning, metadata, tagging, syncing with existing buckets and more
+      </h1>
+    </div>
+    <div
+      class="flex justify-content-center mb-3"
+      style="height: 32px"
+    >
+      <p class="text-xl">
+        We continue to expand the features of BCBox. Stay tuned for documentation on our wiki.<br /><br />
+      </p>
+    </div>
+    <div class="flex justify-content-center mb-8 text-xl">
+      <ul>
+        <li>
+          Need a bucket? Contact <a href="mailto: NRIDS.Optimize@gov.bc.ca">NRIDS Optimization</a>
+          (Natural Resource ministries) or your <br /> ministry's service desk
+        </li>
+        <li>
+          See BCBox on
+          <a
+            href="https://github.com/bcgov/bcbox"
+            target="_blank"
+          >Github</a>
+        </li>
+        <li>
+          Learn more about the
+          <a
+            href="https://bcgov.github.io/common-service-showcase/"
+            target="_blank"
+          >Common Services Showcase Team</a>
+        </li>
+        <li>
+          Email the <a href="mailto: nrm.commonserviceshowcase@gov.bc.ca">Common Services Showcase Team</a>
+          with questions or feedback
+        </li>
+      </ul>
+    </div>
+    <div class="flex justify-content-center mb-8">
+      <h3 v-if="!getIsAuthenticated">
+        <Button
+          @click="login()"
+        >
+          Log in to get started
+        </Button>
+      </h3>
+      <h3 v-else>
+        <Button
+          @click="goToBuckets()"
+        >
+          Go to My Buckets
+        </Button>
+      </h3>
```

Comment:
The button action for both of these should be the same: goToBuckets(). We shouldn't be permuting its behavior here because we should be depending on underlying infra to do the login for us if necessary. The only thing that should be changing here is the flavor text, and that can be a simple computed based on whether you're actually authenticated or not.

Another note is that for link navigation instances like this, we should be leveraging `<router-link>` instead. This could be collapsed down to something like the following IIRC:
```vue
<router-link to="/list/buckets">
  <Button>
    {{ !getIsAuthenticated ? 'Log in to get started' : 'Go to My Buckets' }}
  </Button>
</router-link>
```